lib/game.rb

```diff
@@ -1 +1,23 @@
-puts "Welcome to Tic-Tac-Toe!"
\ No newline at end of file
+require_relative "./presenter"
+require_relative "./displayer"
+class Game
+  attr_reader :display
+  def initialize(display, presenter)
+    @display = display
+    @presenter = presenter
+  end
+
+  def start
+    @display.output("Welcome to Tic Tac Toe!")
+  end
+
+  def play
+    @display.output(@presenter.print_board)
+  end
+end
+
+p = Presenter.new
+d = Displayer.new($stdout)
+g = Game.new(d, p)
+g.start
+g.play
```

Comment:
I recommend to use meaningful names here to make your code more expressive.  Example, game.start and game.play is a lot more expressive than g.start, g.play.  Also, d and p don't have meaning to me.

Reply:
Names changed for readability